Makefile.centos

```diff
@@ -0,0 +1,19 @@
+include Makefile
+
+SERVER_SRC_FILE:=$(SERVER_DIR)/Containerfile.centos
+AD_SERVER_SRC_FILE:=$(AD_SERVER_DIR)/Containerfile.centos
```

Comment:
I'd rather not add any overrides that don't exist yet. So for example  images/ad-server/Containerfile.centos doesn't exist... so it's never going to do anything useful.
I'd be OK if you left some of these variable names in but commented them out, just to serve as breadcrumbs.

Reply:
I changed the commit to only add the server centis build flavor, but it still adds overrides to files that don't exist (yet) ...  we could add those in the future if we want to.